_posts/2022-12-05-Introducing-Identity.markdown

```diff
@@ -0,0 +1,40 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  
+
+Within the OpenSearch Project, the Security plugin allows finer levels of access control providing a more satisfying answer.  Security uses the plugin architecture of OpenSearch that forces concepts around identity to be optional in core, plugins, and extensions components.  Optional security is not enough for critical data, we will do to do better.
+
+Let’s permeate identity in every aspect of OpenSearch – enabling security by default - everywhere.
+
+## Identity is more than a user
+In this blog post we use the term  *Subject* that includes users and non-person entities.  Following NIST's defintion Subjects are '[a]n individual, process, or device that causes information to flow among objects or change to the system state.'
+
+Among the OpenSearch-Project's scenario there is monitoring, data management, replication, machine learning, and notificatios.  The identities of thse systems and process need a common description as a Subject.

```

Comment:
The term Subject is introduced here. But the term doesn't appear again in subsequent sections of the blog. Is it important to include it if it won't be used later on in this blog? If it is an integral part of understanding "identity" it would be good to draw a clearer connection between the two. Are they synonymous?

Reply:
Removed the term completely, its better suite in a more technical context